guides/common/modules/con_configuring-an-ldap-server-as-an-external-identity-provider-for-project.adoc

```diff
@@ -7,9 +7,17 @@
 Lightweight Directory Access Protocol (LDAP) is a set of open protocols used to access centrally stored information over a network.
 With {Project}, you can use one or multiple LDAP directories for external authentication.
 
+[NOTE]
+====
+While you can configure the LDAP server integrated with {FreeIPA} as an external authentication source, {FreeIPA} users will not be able to log in using single sign-on.
+Instead, consider configuring {FreeIPA} as an external identity provider.
```

Comment:
"consider" is kinda weak. Can we make it more opinionated? Are there any benefits from doing this? Is there any kind of official recommendation?

Also, starting line 12 with "While" feels off to me.